pkg/wellknownnamespaces/util.go

```diff
@@ -20,7 +20,7 @@ func IsCentOSNamespace(namespace string) bool {
 // The namespace is expected to be of form `namespacename:version` but only `namespacename` is required.
 // For example: rhel:7, rhel:8, centos:8, ubuntu:14.04.
 func IsRHELNamespace(namespace string) bool {
-	return strings.HasPrefix(namespace, "rhel")
+	return (strings.HasPrefix(namespace, "rhel") || strings.HasPrefix(namespace, "rhcos"))
```

Comment:
So we are sure that `rhcos` is the correct string that we will get here? 
Could we simply return `rhel` from the regexp and skip this modification?

I am not suggesting anything, just asking as I truly don't know myself :) @RTann help :)

Reply:
IIRC, we ended our sync meeting yesterday (Thursday) with the notion that we want to identify RHCOS as `rhcos:4.11`, hence this check, to ensure we identify it as discussed and still enter the verified scanning functionality.
But I'm open to changing this should I have misunderstood that :)

Reply:
I'm thinking we make a separate `IsRHCOSNamespace` and then do the `||` when used by the client. What do you think?